starknet-replay/src/runner/replay_statistics.rs

```diff
@@ -55,4 +61,83 @@ impl ReplayStatistics {
                 .or_insert(*weight);
         }
     }
+
+    /// Returns the number of different concrete libfunc names in the
+    /// `ReplayStatistics` object.
+    pub fn get_number_of_libfuncs(&self) -> usize {
+        self.concrete_libfunc.len()
+    }
+
+    /// Returns the number of calls of the most frequently called concrete
+    /// libfunc.
+    ///
+    /// It returns `None` if the map of libfuncs is empty.
+    pub fn get_highest_frequency(&self) -> Option<usize> {
```

Comment:
I am wondering about functions to modify the data in this container. Would that ever be useful?